u8timeseries/timeseries.py

```diff
@@ -743,6 +743,48 @@ def update(self,
                 new_hi.update(conf_hi)
             return TimeSeries(new_series, new_lo, new_hi)
 
+    def upsample(self, freq: str, method: str = 'pad') -> 'TimeSeries':
```

Comment:
what happens if this is called with a lower frequency than the original series? Will this downsample?

Reply:
It will throw an exception

Reply:
oh yes sorry missed it :) but how about also supporting downsampling, and renaming this one to `resample()` ?